src/javascript/app/common/contract_manager.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable class-methods-use-this */
+import { triggerContractChange } from '../hooks/events';
+
+class ContractManager {
+    constructor() {
+        if (!window.contract) {
+            window.contract = {};
```

Comment:
please explain why was this approach selected to store contract data in the global `window` object? same for `window.trade`

Reply:
These data are only available in the current session, so upon browser refresh we want these data to go away. We are leveraging on global variables to observe and fully use the state features of react.

Reply:
But if you want these data to do away on browser refresh, why not sessionStorage in this case?

Reply:
session storage doesn't go away after refresh.

Reply:
@prince-deriv seems the `contract` object is only used in this file. Is there any reason we should set it in window?
Can we set it as a class member object? or, use a separate context object just for `contract`?